networkx/classes/tests/test_function.py

```diff
@@ -16,22 +17,22 @@ def setUp(self):
         self.DGedges=[(0,1),(0,2),(0,3),(1,0),(1,1),(1,2)]
 
     def test_nodes(self):
-        assert_equal(list(self.G.nodes()),list(nx.nodes(self.G)))
-        assert_equal(list(self.DG.nodes()),list(nx.nodes(self.DG)))
+        assert_nodes_equal(self.G.nodes(),list(nx.nodes(self.G)))
+        assert_nodes_equal(self.DG.nodes(),list(nx.nodes(self.DG)))
 
     def test_edges(self):
-        assert_equal(list(self.G.edges()),list(nx.edges(self.G)))
-        assert_equal(list(self.DG.edges()),list(nx.edges(self.DG)))
-        assert_equal(list(self.G.edges(nbunch=[0,1,3])),list(nx.edges(self.G,nbunch=[0,1,3])))
-        assert_equal(list(self.DG.edges(nbunch=[0,1,3])),list(nx.edges(self.DG,nbunch=[0,1,3])))
+        assert_edges_equal(self.G.edges(),list(nx.edges(self.G)))
+        assert_equal(sorted(self.DG.edges()),sorted(nx.edges(self.DG)))
```

Comment:
No, `DG` is a DiGraph. `assert_edges_equal` considers `(u, v)` and `(v, u)` to be equivalent.